runtime/doc/lua.txt

```diff
@@ -3351,4 +3351,91 @@ totable({f}, {...})                                       *vim.iter.totable()*
     Return: ~
         (table)
 
+
+==============================================================================
+Lua module: net                                                      *lua-net*
+
+fetch({url}, {opts})                                         *vim.net.fetch()*
+    Perform an asyncronous network request.
+
+    Example: >lua
+    vim.net.fetch("https://example.com/api/get", {
+      on_complete = function (response)
```

Comment:
```suggestion
      on_result = function (response)
```

or `on_done`. 

Looks like [nodejs](https://nodejs.org/api/net.html#netcreateconnectionoptions-connectlistener) calls this `on_end` , are there other conventions?